### PR TITLE
Expose mouse_moves

### LINKED ISSUE #49: Expose mouse moves from InputDiff
There is no easy way to get mouse moves (e.g. delta of position since last update) atm.
However, we have all capabilities to easily do that by adding new method to `Input` impl.
cc @shmutalov

### PR DESCRIPTION
**Pull request by vitvakatu**
Fixes #49

### CODE REVIEW

File: src/input.rs [hunk 1]
```diff
@@ -70,6 +70,19 @@ impl Input {
         self.0.mouse_pos
     }
 
+    pub fn get_mouse_movements(&self) -> Vec<mint::Vector2<f32>> {
```
Comment (kvark) [inline src/input.rs]: should return `&[]` without cloning

**Comment by kvark** [comment]
Please rebase and squash

**Comment by vitvakatu** [comment]
@kvark ping

**Pull request merged**